packages/mpp/mpp.0.3.2/findlib

```diff
@@ -0,0 +1 @@
+mpp
```

Comment:
This file has no longer a meaning, you can safely remove it

Reply:
OK, done. :)